src/Augurk/Filters/NoCacheHeaderFilter.cs

```diff
@@ -28,24 +27,27 @@ public class NoCacheHeaderFilter : ActionFilterAttribute
         /// <summary>
         /// Adds or alters the <see cref="CacheControlHeaderValue"/> on the response headers to ensure the repsonse will not be cached.
         /// </summary>
-        /// <param name="actionExecutedContext">The current context.</param>
-        public override void OnActionExecuted(HttpActionExecutedContext actionExecutedContext)
+        /// <param name="context">The current context.</param>
+        public override void OnActionExecuted(ActionExecutedContext context)
```

Comment:
Still need to figure out if we still need this.

Reply:
Perhaps not. There was one particular browser which had a very nasty habit of caching all of this... that browser is dying at the moment....
So even if it is not supported out of the box, we might be able to forego it.

Reply:
I'll delete it for now then. I think might want to have a look at a more efficient way of caching responses from the API anyway. Perhaps we could do something with ETags or something.